Automattic/wp-calypso
# Stats: Fixing module size for Authors and Search terms

Related to #90252 

## Proposed Changes

* fix module width for a large breakpoint (~1000px)
* apply the update to the Authors and Search terms modules

## Testing Instructions



* open the live branch
* resize the browser to 1000px width
* check that all modules are aligned properly, especially Authors for Calypso and Search terms for Jetpack (not like on the screenshot below)
* experiment with disabling the Videos module

Before:
<img width="714" alt="SCR-20240510-oogk" src="https://github.com/Automattic/wp-calypso/assets/112354940/594f1e45-6988-44d9-bf7b-37154110bb4e">


## Pre-merge Checklist
